server/controllers/refer/referralsController.test.ts

```diff
@@ -36,8 +33,15 @@ describe('ReferralsController', () => {
   const courseOffering = courseOfferingFactory.build()
 
   let referralsController: ReferralsController
+  let request: DeepMocked<Request>
```

Comment:
Just seen you've more or less done the same in #195 @yndajas.  Will wait for that to be merged then rebase.

Reply:
Ah! I see you reset `request` and `next` too - I don't think we ever alter those, but it's probably a good idea in case we do?